pkg/minishift/addon/manager/addon_manager.go

```diff
@@ -358,29 +358,36 @@ func setStateAndPriority(addOn addon.AddOn, configMap map[string]*config.AddOnCo
 
 func compareComponentVersions(currentVersion, requiredVersion string, componentName string, versionRange string) error {
 	if strings.HasPrefix(requiredVersion, ">=") {
-		// This will work for both upstream and downstream.
-		if util.VersionOrdinal(currentVersion) < util.VersionOrdinal(strings.TrimPrefix(requiredVersion, ">=")) {
+		currentVersion, _ := semver.Make(currentVersion)
+		requiredVersion, _ := semver.Make(strings.TrimPrefix(requiredVersion, ">="))
+		if currentVersion.LT(requiredVersion) {
```

Comment:
Maybe it would be more readable if the if-statement was not to catch error, but to catch valid case, so for greater or equal, it would use currentVersion.GE():

```
	if strings.HasPrefix(requiredVersion, ">=") {
		currentVersion, _ := semver.Make(currentVersion)
		requiredVersion, _ := semver.Make(strings.TrimPrefix(requiredVersion, ">="))
		if currentVersion.GE(requiredVersion) {
			return nil
		}
		return fmt.Errorf("\nAdd-on does not support %s version %s. "+
			"You need to use a version %s", componentName, currentVersion, versionRange)
}
```
or
```
if currentVersion.GE(requiredVersion) != true {
	return fmt.Errorf("\nAdd-on does not support %s version %s. "+
		"You need to use a version %s", componentName, currentVersion, versionRange)
}
```